pkgs/by-name/ni/nibtools/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, opencbm
+, cc65
+}:
+
+stdenv.mkDerivation {
+  pname = "nibtools";
+  version = "unstable-2024-02-22";
```

Comment:
please prefix the version with a 0 or the last stable version

Reply:
There is no release version.

Reply:
Then please prepend a 0, there are new conventions https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming. This is essentially to fix `lib.getVersion` which splits at the first number